client/my-sites/plan-features-2023-grid/components/billing-timeframe.tsx

```diff
@@ -74,82 +73,69 @@ function usePerMonthDescription( {
-								comment: 'Excl. Taxes is short for excluding taxes',
-							}
-					  )
-					: translate( 'per month, %(fullTermDiscountedPriceText)s for the first year', {
-							args: { fullTermDiscountedPriceText },
-					  } );
-			}
-
-			if ( PLAN_BIENNIAL_PERIOD === billingPeriod ) {
-				return displayNewPriceText
-					? translate(
-							'per month, %(fullTermDiscountedPriceText)s for the first year, Excl. Taxes',
-							{
-								args: { fullTermDiscountedPriceText, rawPrice },
-								comment: 'Excl. Taxes is short for excluding taxes',
-							}
-					  )
-					: translate( 'per month, %(fullTermDiscountedPriceText)s for the first year', {
-							args: { fullTermDiscountedPriceText },
-					  } );
-			}
-		} else if ( rawPrice ) {
-			if ( PLAN_ANNUAL_PERIOD === billingPeriod ) {
-				return displayNewPriceText
-					? translate( 'per month, %(rawPrice)s billed annually, Excl. Taxes', {
-							args: { rawPrice },
-							comment: 'Excl. Taxes is short for excluding taxes',
-					  } )
-					: translate( 'per month, %(rawPrice)s billed annually', {
-							args: { rawPrice },
-					  } );
-			}
-
-			if ( PLAN_BIENNIAL_PERIOD === billingPeriod ) {
-				return displayNewPriceText
-					? translate( 'per month, %(rawPrice)s billed every two years, Excl. Taxes', {
-							args: { rawPrice },
-							comment: 'Excl. Taxes is short for excluding taxes',
-					  } )
-					: translate( 'per month, %(rawPrice)s billed every two years.', {
-							args: { rawPrice },
-					  } );
-			}
+	const discountedPrice = planPrices.discountedPrice;
+	const fullTermDiscountedPriceText =
+		currencyCode && discountedPrice
+			? formatCurrency( discountedPrice, currencyCode, { stripZeros: true } )
+			: null;
+	const rawPrice =
+		currencyCode && planPrices.originalPrice
+			? formatCurrency( planPrices.originalPrice, currencyCode, { stripZeros: true } )
+			: null;
+
+	if ( fullTermDiscountedPriceText ) {
+		if ( PLAN_ANNUAL_PERIOD === billingPeriod ) {
+			return translate(
+				'per month, %(fullTermDiscountedPriceText)s for the first year, Excl. Taxes',
+				{
+					args: { fullTermDiscountedPriceText },
+					comment: 'Excl. Taxes is short for excluding taxes',
+				}
+			);
+		}
+
+		if ( PLAN_BIENNIAL_PERIOD === billingPeriod ) {
+			return translate(
+				'per month, %(fullTermDiscountedPriceText)s for the first two years, Excl. Taxes',
+				{
+					args: { fullTermDiscountedPriceText },
+					comment: 'Excl. Taxes is short for excluding taxes',
+				}
+			);
+		}
+
+		if ( PLAN_TRIENNIAL_PERIOD === billingPeriod ) {
+			return translate(
+				'per month, %(fullTermDiscountedPriceText)s for the first three years, Excl. Taxes',
+				{
+					args: { fullTermDiscountedPriceText },
+					comment: 'Excl. Taxes is short for excluding taxes',
+				}
+			);
+		}
+	} else if ( rawPrice ) {
+		if ( PLAN_ANNUAL_PERIOD === billingPeriod ) {
+			return translate( 'per month, %(rawPrice)s billed annually, Excl. Taxes', {
```

Comment:
`Non-blocking / Nit `:

I'm noticing a distinct difference in language that we use

`billed annually / billed every two years / billed every three years`

vs. 

`for the first year / for the first two years / for the first three years`

This seems fine to me, but I wanted to make sure that this variation is purposeful.

Reply:
Good spot! It is purposeful as we want to make the distinction that the discounted price applies for the first year/term only, and then the full price of the plan would apply for subsequent terms.